web/src/pages/people/index.tsx

```diff
@@ -44,6 +54,33 @@ export default function PeoplePage() {
   const isAdmin = userData?.roles.includes("admin");
   const isIAO = userData?.roles.includes("information-asset-owner");
   const isTreOpsStaff = userData?.roles.includes("tre-ops-staff");
+  const canSearch = isTreOpsStaff || isAdmin;
+
+  const handleUserSearch = async (query: string) => {
+    if (query.length < 3) {
+      setSearchErrorMessage("minimum 3 characters required to perform search");
+    } else {
+      if (query === "") {
+        setUsers(originalUsers);
+      } else {
+        const response = await getUsers({ query: { find: query } });
+        if (response.response.ok && response.data) {
+          setUsers(response.data);
+          setSearchErrorMessage("");
+        } else {
+          setSearchErrorMessage("Oops, something went wrong with your search. Refresh and try again");
+        }
+      }
+    }
+  };
```

Comment:
```suggestion
 const handleUserSearch = async (query: string): Promise<void> => {
  if (query.length < 3) {
    setSearchErrorMessage("minimum 3 characters required to perform search");
    return;
  }

  if (query === "") {
    setUsers(originalUsers);
    return;
  }

  const response = await getUsers({ query: { find: query } });

  if (!response.response.ok || !response.data) {
    setSearchErrorMessage("Oops, something went wrong with your search. Refresh and try again");
    return;
  }

  setUsers(response.data);
  setSearchErrorMessage("");
};

```

I find nested `if` blocks hard to read. Using early returns makes the logic easier to follow, at least for me.